LandSandBoat/server
# [SQL][Items][Additional effects] Item mods 17041-17483

**_I affirm:_**

## What does this pull request do?

Adds additional effect item mods to items 17041-17483.

## Steps to test these changes

As GM, !additem 17041|17083|17106|17305|17306|17313|17314|17315|17316|17322|17323|17324|17325|17411|17483
Equip item and hit things until you see the effect take place.
